packages/cli/src/commands/run.js

```diff
@@ -86,7 +87,8 @@ export const run = async ({
             args.unshift(filePath);
           }
 
-          const readableP = E(agent).lookup(bundleName);
+          const bundleNamePath = bundleName && parsePetNamePath(bundleName);
+          const readableP = E(agent).lookup(...bundleNamePath);
```

Comment:
We're in an `if` clause where `bundleName !== undefined`:
```suggestion
          const readableP = E(agent).lookup(...parsePetNamePath(bundleName));
```

Reply:
Nice catch.